src/tree/ActionTreeItem.ts

```diff
@@ -18,7 +18,7 @@ import { JobTreeItem } from './JobTreeItem';
 export class ActionTreeItem extends AzExtParentTreeItem implements IAzureResourceTreeItem {
     public static contextValueCompleted: string = 'azureStaticActionCompleted';
     public static contextValueInProgress: string = 'azureStaticActionInProgress';
-    public parent: ActionsTreeItem;
+    public parent!: ActionsTreeItem;
```

Comment:
![image](https://user-images.githubusercontent.com/36966225/141853116-0a466365-f55a-4e9a-a488-6a3b709daf8b.png)

Reply:
(I know what the `!` does but it's always funny to imagine that we're literally just yelling the variable name louder to drown out TS' complaints)